src/mbgl/gl/value.hpp

```diff
@@ -259,7 +259,7 @@ struct RasterPos {
         double z;
         double w;
     };
-    static const constexpr Type Default = { 0, 0, 0, 0 };
+    static const constexpr Type Default = { 0, 0, 0, 1 };
```

Comment:
According to the [documentation](https://www.opengl.org/sdk/docs/man2/xhtml/glRasterPos.xml), this is the actual default.